frontend/src/app/api/calendarClient.tsx

```diff
@@ -1,37 +1,56 @@
-import axios, { AxiosInstance } from "axios";
+import axios, { AxiosInstance } from 'axios';
 type UUID = string;
 
 export interface CreateEventRequest {
-    userId: UUID;
-    startAt: Date;
-    endAt: Date;
-    eventText: string;
+  userId: UUID;
+  startAt: Date;
+  endAt: Date;
+  eventText: string;
 }
 
 export interface Event {
-    id: UUID;
-    userId: UUID;
-    startAt: Date;
-    endAt: Date;
-    eventText: string;
+  id: UUID;
+  userId: UUID;
+  startAt: Date;
+  endAt: Date;
+  eventText: string;
+}
+
+export type ByDateResponse = Record<string, Event[]>;
+
+function formatDate(date: Date): string {
+  const offset = date.getTimezoneOffset()
+  const offsetApplied = new Date(date.getTime() - (offset*60*1000))
+  return offsetApplied.toISOString().split('T')[0]
 }
 
 class CalendarClient {
-    private axiosInstance: AxiosInstance;
-
-    constructor(baseURL: string) {
-        this.axiosInstance = axios.create({
-          baseURL,
-          headers: {
-            "Content-Type": "application/json",
-          },
-        });
-    }   
-
-    async createEvent(data: CreateEventRequest): Promise<Event> {
-        const response = await this.axiosInstance.post<Event>("/events", data);
-        return response.data;
-    }
+  private axiosInstance: AxiosInstance;
+
+  constructor(baseURL: string) {
+    this.axiosInstance = axios.create({
+      baseURL,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+  }
+
+  async createEvent(data: CreateEventRequest): Promise<Event> {
+    const response = await this.axiosInstance.post<Event>('/events', data);
+    return response.data;
+  }  
+
+  async displayEvents(userId: UUID, fromDate: Date, toDate: Date) {
```

Comment:
I'd name this function differently, as it is not 'displaying' anything. fetchEvents, getEvents, something along these lines I thing would be better.